aptos-move/framework/aptos-framework/sources/event.spec.move

```diff
@@ -7,7 +7,7 @@ spec aptos_framework::event {
     spec emit_event {
         pragma opaque;
         aborts_if [abstract] false;
-        ensures handle_ref.counter == old(handle_ref.counter) + 1;
+        ensures handle_ref.counter < MAX_U64 ==> handle_ref.counter == old(handle_ref.counter) + 1;
```

Comment:
Do we need to add the condition `handle_ref.counter < MAX_U64` although it can be proved without it?

Reply:
Yes, there is an assume clause in the `emit_event` function. Without this condition, there would be an inconsistency issue.